components/catalogs/back/src/Application/Persistence/DisableCatalogsQueryInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Catalogs\Application\Persistence;
+
+/**
+ * @author    Willy Mesnage <willy.mesnage@akeneo.com>
+ * @copyright 2022 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+interface DisableCatalogsQueryInterface
+{
+    /**
+     * @param array<string> $catalogsUUID
+     */
+    public function execute(array $catalogsUUID): void;
```

Comment:
```suggestion
    public function execute(array $id): void;
```
As it is done for other queries related to catalog directly